docs/zh/latest/building-apisix.md

```diff
@@ -141,15 +103,6 @@ tar -xvf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz && \
 nohup etcd >/tmp/etcd.log 2>&1 &
 ```
 
-</TabItem>
-
-<TabItem value="mac">
-
-```shell
-brew install etcd
-brew services start etcd
-```
-
```

Comment:
we may need save this one, user can build and run apsix at docker, he will need install etcd by brew

Reply:
I think if you build with Docker, you can also run etcd with Docker. Wouldn't it be better to put Docker's documentation together?

Reply:
cc @kayx23  how do u think about this

Reply:
What does this doc have to do with Docker?

Reply:
So the change to me looks like, going forward:

* to build APISIX from source with everything installed on the host, only Linux is supported; details are in [this doc](https://apisix.apache.org/docs/apisix/next/building-apisix/)
* to build APISIX from source in Docker, it is OS agnostic; details are in [this doc](https://apisix.apache.org/docs/docker/build/#build-an-image-from-customizedpatched-source-code)

In that case, I agree with @monkeyDluffy6017 that the relevant etcd installation supplying building APISIX in Docker should go in the Docker doc for clarity. As you can see in the last comment, it also took me a second to figure out what the connection was between macOS etcd installation and building in Docker, as two docs are separate.